zotbins/ZotbinsCore
# Added script to create header from .env

## Description

Added script to create header from .env to store secret information. Closes #22

In ZotBins Core, we need to include bin specific information that we do not want to commit to the repo like WiFi passwords or bin height. This PR uses the values from .env to create config.hpp with the relevant information, so we can put the WiFi password in our code without accidentally committing it to Github.

## Type of change


## Testing and Verification Instructions

1. Create `.env` with `.env.txt` as a template
2. Build the project
3. Verify the project compiles and `config.hpp` is created.

## Checklist:
